tests/integration/api/test_activation.py

```diff
@@ -362,51 +362,32 @@ def test_disable_activation(
 
 @pytest.mark.django_db
 def test_list_activation_instances(
-    default_activation: models.Activation, client: APIClient
+    default_activation: models.Activation,
+    default_activation_instances: List[models.RulebookProcess],
+    client: APIClient,
 ):
-    instances = models.RulebookProcess.objects.bulk_create(
-        [
-            models.RulebookProcess(
-                name="test-activation-instance-1",
-                activation=default_activation,
-                git_hash=PROJECT_GIT_HASH,
-            ),
-            models.RulebookProcess(
-                name="test-activation-instance-1",
-                activation=default_activation,
-                git_hash=PROJECT_GIT_HASH,
-            ),
-        ]
-    )
+    instances = default_activation_instances
     response = client.get(
         f"{api_url_v1}/activations/{default_activation.id}/instances/"
     )
     data = response.data["results"]
     assert response.status_code == status.HTTP_200_OK
     assert len(data) == len(instances)
-    assert data[0]["name"] == instances[0].name
-    assert data[1]["name"] == instances[1].name
+    # invert the indices because response data is descending by default
+    assert data[0]["name"] == instances[1].name
+    assert data[1]["name"] == instances[0].name
```

Comment:
Performing a suitable sort on both lists will insulate this test from any future changes that affect response content ordering.

Reply:
Good point. I will add sorting for both lists in the test.